app/controllers/concerns/vendor_api/api_validations_and_error_handling.rb

```diff
@@ -0,0 +1,54 @@
+module VendorAPI
+  module APIValidationsAndErrorHandling
+    extend ActiveSupport::Concern
+
+    included do
+      before_action :validate_metadata!
+
+      rescue_from ValidationException, with: :render_validation_error
```

Comment:
I think we have this logic in the `VendorAPIController`?

Reply:
ValidationExceptions are only raised when calling different services, so I think this doesn't belong in the main controller (as not all API controllers call services)

Reply:
discussed in person: there is similar logic in the vendor API controller, think about including this concern there

Reply:
@cwrw as we don't want to use this in the DecisionsController now, I assume we should keep it separate?